src/EducationService.Mappers/Models/UserEducationDataMapper.cs

```diff
@@ -7,21 +7,36 @@ namespace LT.DigitalOffice.EducationService.Mappers.Models
 {
   public class UserEducationDataMapper : IUserEducationDataMapper
   {
+    private readonly IEducationFormDataMapper _educationFormDataMapper;
+    private readonly IEducationTypeDataMapper _educationTypeDataMapper;
+
+    public UserEducationDataMapper(
+      IEducationFormDataMapper educationFormDataMapper,
+      IEducationTypeDataMapper educationTypeDataMapper)
+    {
+      _educationFormDataMapper = educationFormDataMapper;
+      _educationTypeDataMapper = educationTypeDataMapper;
+    }
+
     public EducationData Map(DbUserEducation dbUserEducation)
     {
       if (dbUserEducation is null)
       {
         return null;
       }
 
+      EducationFormData formData = _educationFormDataMapper.Map(dbUserEducation.EducationForm);
```

Comment:
you can do not create the variable and write educationForm: _educationFormDataMapper.Map(dbUserEducation.EducationForm)